Example/OriginateUI/CircleViewController.m

```diff
@@ -0,0 +1,89 @@
+//
+//  CircleViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 7/7/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "CircleViewController.h"
+#import "CodeViewController.h"
+@import OriginateUI;
+
+@interface CircleViewController ()
+
+@property (nonatomic, strong) UILabel *mainLabel;
+@property (nonatomic, strong) UILabel *secondLabel;
+
+@end
+
+@implementation CircleViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.view.backgroundColor = [UIColor whiteColor];
+    CGFloat screenHeight = CGRectGetHeight(self.view.frame);
+    CGFloat screenWidth = CGRectGetWidth(self.view.frame);
```

Comment:
UI geometry-related initialization in `viewDidLoad` is generally meaningless:p Because the view frame is not set in `viewDidLoad`, whatever information of the frame is not the final one loaded on the screen. If you want to size the subviews based on the size of the controller's root view, it's easier to use autolayout by adding constraints in `viewDidLoad`, then the system would assign frames to the subviews automatically based on the constraints u set here.